src/interface/guide/index.tsx

```diff
@@ -269,23 +273,18 @@ export function useAnalyzer<T extends typeof Module>(value: string | T) {
  * }
  * ```
  */
-export function useAnalyzers<T extends typeof Module>(moduleTypes: T[]): InstanceType<T>[];
-export function useAnalyzers(moduleKeys: string[]): Module[];
-export function useAnalyzers<T extends typeof Module>(values: (string | T)[]) {
+export function useAnalyzers<Arr extends { [Key: number]: typeof Module }>(
+  values: Arr,
+): ModuleList<Arr> {
```

Comment:
nobody was using string-keyed `useAnalyzers` calls, and this modification makes the result a tuple type instead of an array of unions. for example:

```ts
// old
useAnalyzers([DeathStrike, RunicPowerTracker]) // -> (DeathStrike | RunicPowerTracker)[];
// new
useAnalyzers([DeathStrike, RunicPowerTracker]) // -> [DeathStrike, RunicPowerTracker]
```